contracts/DecentralizedKV.sol

```diff
@@ -13,12 +13,14 @@ contract DecentralizedKV is OwnableUpgradeable {
         PaddingPer31Bytes
     }
 
-    uint256 public storageCost; // Upfront storage cost (pre-dcf)
+    // Upfront storage cost (pre-dcf)
+    uint256 public constant storageCost = 1500000000000000; // storageCost - 1,500,000Gwei forever per blob - https://ethresear.ch/t/ethstorage-scaling-ethereum-storage-via-l2-and-da/14223/6#incentivization-for-storing-m-physical-replicas-1
```

Comment:
I would suggest using a base class (EthStorageConstStorage?) to declare all those const, like this: https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/core/EigenDAServiceManagerStorage.sol